classes/ValidFormBuilder/Base.php

```diff
@@ -956,6 +956,12 @@ protected function __setMeta($property, $value, $blnOverwrite = false)
 
             // Make sure no empty values are left in the array.
             $arrMeta = array_filter($arrMeta);
+
+            // Make sure there are no duplicate entries for the 'class' property
+            if ($property === 'class') {
```

Comment:
I would add a `strtolower` call around the `$property` variable. Strings are tricky...